indigo_app/static/stylesheets/_tasks-view.scss

```diff
@@ -0,0 +1,39 @@
+// helper classes
+.task-list-card-full-height {
+  height: 100%;
+  max-height: 70vh;
```

Comment:
This works better than I expected - how about making it 80vh?

Reply:
Updated